.github/workflows/code-quality.yml

```diff
@@ -42,3 +42,9 @@ jobs:
 
       - name: Check Python formatting
         run: black --check .
+
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
```

Comment:
[consider] `@master`?

Reply:
Ah I didn't even think about that, just copied from the sample on the actions docs page. Github must do something to convert `master` to `main` then, because it works. Guess this needs to be changed everywhere, annoyingly including the one that just got merged into openassetio mainline.

Reply:
Yup, I should have picked up on that in the original review, apologies.